.github/workflows/00_github_workflows_staging.yml

```
@@ -1,41 +1,19 @@
+name: 00 Staging Check
 
 on:
   pull_request:
-    paths:
-      - 'staging/**'
-  workflow_dispatch:
+    types: [opened, reopened, edited, synchronize, review_requested]
+
+concurrency:
+  group:modules-${{github.event.pull_request.number}}
+  cancel-in-progress: true
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 9: mapping values are not allowed in this context

(syntax-check)